server/src/main/java/org/opensearch/ingest/ConfigurationUtils.java

```diff
@@ -67,6 +67,7 @@ public final class ConfigurationUtils {
 
     public static final String TAG_KEY = "tag";
     public static final String DESCRIPTION_KEY = "description";
+    public static final String IGNORE_FAILURE_KEY = "ignore_failure";
```

Comment:
Now that you've added this constant, can you please update the `readProcessor` method in this class (used for Ingest processors) to use it too? It currently has the string `"ignore_failure"` hard-coded.